lustrefs-exporter/src/brw_stats.rs

```diff
@@ -423,9 +441,36 @@ pub fn build_target_stats(
         TargetStats::ThreadsMax(_x) => {}
         TargetStats::ThreadsStarted(_x) => {}
         TargetStats::RecoveryStatus(_x) => {}
```

Comment:
Could we provide implementation for this one? It will handy to provide information/alert when a target is in recovery